src/app/components/Clipboard/Clipboard.test.tsx

```diff
@@ -33,6 +35,30 @@ describe("Clipboard", () => {
 		expect(asFragment()).toMatchSnapshot();
 	});
 
+	it("should change the tooltip content when clicked", async () => {
+		const { baseElement, getByTestId } = render(
+			<I18nextProvider i18n={i18n}>
+				<Clipboard>
+					<span>Hello!</span>
+				</Clipboard>
+			</I18nextProvider>,
+		);
+
+		act(() => {
+			fireEvent.mouseEnter(getByTestId("clipboard__wrapper"));
+		});
+
+		expect(baseElement).toHaveTextContent("Copy to clipboard");
+		expect(baseElement).not.toHaveTextContent("Copied!");
```

Comment:
Should probably use the `CLIPBOARD.COPIED` key here instead of hardcoding text, same for the `Copy to clipboard`.

Reply:
I think there's multiple occurrences of such checks across the test suite, probably worthy of their own PR. I'll fix this once i'm back at the desk.